### PR TITLE
Show resource usage on grainlist page

### PR DESCRIPTION
**Pull request by zarvox**

### CODE REVIEW

**Comment by zarvox** [comment]
Don't merge yet; I'm going to change this up a bit.

**Comment by zarvox** [comment]
Scratch that; the initial version does what I wanted.

**Comment by zarvox** [comment]
Garply, retest this please

File: shell/packages/sandstorm-ui-grainlist/grainlist-client.js [hunk 1]
```diff
@@ -80,11 +80,19 @@ Template.sandstormGrainList.helpers({
   filteredSortedGrains: filteredSortedGrains,
   searchText: function() {
     return Template.instance().data._filter.get();
+  },
+  myGrainsCount: function () {
+    return Template.instance().data._db.currentUserGrains({}, {}).fetch().length;
```
Comment (kentonv) [inline shell/packages/sandstorm-ui-grainlist/grainlist-client.js]: Please use count(); it's much faster than fetch().length.

File: shell/packages/sandstorm-ui-grainlist/grainlist-client.js [hunk 2]
```diff
@@ -80,11 +80,19 @@ Template.sandstormGrainList.helpers({
...
+  },
+  myGrainsSize: function () {
+    // TODO(cleanup): extract prettySize and other similar helpers from globals into a package
+    return prettySize(Template.instance().data._db.getMyUsage().storage);
```
Comment (kentonv) [inline shell/packages/sandstorm-ui-grainlist/grainlist-client.js]: Please use `Meteor.user().storageUsage`, which is reactive. getMyUsage is kind of broken and I think I'm going to remove it.

**Pull request merged**